src/current/_includes/v23.1/sidebar-data/cloud-deployments.json

```diff
@@ -131,28 +131,28 @@
             ]
```

Comment:
I think "CockroachDB Cloud Security Overview" should just be renamed to "Overview" and the page title should have "Overview" added at the end.

Reply:
"Manage AWS PrivateLink for a Dedicated Cluster" should be "Manage AWS PrivateLink for a Cloud Cluster" or just "Manage AWS PrivateLink" since it could be implied that it's for cloud since it's under "Cloud Deployments". Additionally, the page title should just be "Manage AWS ...." (not "Managing")